### PR TITLE
Week 1

### PR DESCRIPTION
**Pull request by inurzamal**

### CODE REVIEW

File: Week-1/style.css [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  justify-content: space-around;
+  align-items: center;
+  background-color: #f2f2f2;
+}
+
+aside i.fa {
+  font-size: 1.5em;
+}
+
+/* #brand-image{
```
Comment (ShashankSuresh) [inline Week-1/style.css]: Please delete commented out code

File: Week-1/style.css [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+  .music-head img {
+    width: 150px;
+  }
+
+  .catty-music div:nth-child(1) p:first-child {
+    font-size: 1em;
+  }
+
+  /* .catty-music div:nth-child(1) p:not(:first-child) {
```
Comment (ShashankSuresh) [inline Week-1/style.css]: Same here. Delete if not required.

**Comment by ShashankSuresh** [comment]
Attach final output as a screenshot in the description :)

**Comment by inurzamal** [comment]
output

**Review by ShashankSuresh — APPROVED** [review]

**Pull request merged**